integration/tests/cook/test_multi_user.py

```diff
@@ -45,6 +45,9 @@ def test_job_delete_permission(self):
 
     def test_group_delete_permission(self):
         user1, user2 = self.user_factory.new_users(2)
+        with self.user_factory.admin():
```

Comment:
This change seems unrelated to the PR. Split off?

Reply:
Why is this code in this specific location, in this test? This looks to be setup/reset code that belongs somewhere else.

Reply:
Yep, this is unrelated to the default submit pool change. That being said, I saw failures for these tests on this branch, so it seems appropriate to fix them on this branch.

As for why it's here, the resetting needs to happen for the specific user the test is using, which is determined in the test.

Reply:
Thanks for explanation.

Are the test failures caused by the intended code changes in this branch?

Reply:
Nope, they were caused by a test run that was aborted and left a test user's quota in a too-low state.

Reply:
Moved these calls to the `UserFactory`